### PR TITLE
chore: Modify readme.md file

### PR DESCRIPTION
**Pull request by PrayatOsm**
We have added a readme.md file for users who want to know overview and setup of the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
-# dotnet-foundation
\ No newline at end of file
+# Dotnet Foundation
+
+## Overview of the project
+
+Dotnet Foundation is a boiler plate project for ASP .NET projects. This boiler plate can be used in any .NET project for Osmosys. This project contains common files that  can be reused  again and again. Objective of building this project is to make it so that people can copy the project and build on top of it.
```
Comment (sameer-s-b) [inline README.md]: - boiler plate -> boilerplate
- ASP .NET -> ASP.NET.
- Dotnet Foundation is a boilerplate for a ASP.NET WebAPI project.
- Remove "again and again".

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+4. Clone this project.
+5. Open VS Code in this directory `\dotnet-foundation`.
+6. Install the recommended [VS Code extensions](https://imgur.com/XIh4IPI)
+7. In the project's directory `\dotnet-foundation\DotnetFoundation`, execute the following commands to run the project.
+```csharp
+- dotnet restore
+- dotnet build
+- dotnet run --project .\DotnetFoundation.API\
+```
+
```
Comment (sameer-s-b) [inline README.md:21]: Where are the instructions on how to setup the development environment using VS Code.

File: README.md [hunk 3]
```diff
@@ -1 +1,28 @@
...
+
```
Comment (sameer-s-b) [inline README.md:24]: The contributions section is missing. The licenses section is also missing.

File: README.md (hunk 1 above)
Comment (PrayatOsm) [reply README.md]: made all the rectifications as given.

File: README.md (hunk 3 above)
Comment (PrayatOsm) [reply README.md:24]: Added contribution and license section

**Review by sameer-s-b — APPROVED** [review]

**Pull request merged**